src/ActionButton/ActionButton.js

```diff
@@ -10,47 +11,109 @@ export default class WrappedActionButton extends Component {
     color: '#fff',
   }
 
-  renderSub() {
-    let { color, backgroundColor, icon, action } = this.props
+  state = {
+    width: null,
+  }
+
+  handleLayout = ({
+    nativeEvent: {
+      layout: { width },
+    },
+  }) => {
+    if (!this.state.width !== width) {
```

Comment:
Should this be `this.state.width !== width`?

Reply:
Weird how that slipped in there. On it.